validation/views.py

```diff
@@ -67,43 +69,34 @@ def index(request):
             all_phrases = Phrase.objects.exclude(status="auto-validated").order_by(
                 "transcription"
             )
```

Comment:
This suggestion is explained in the next comment.

```suggestion
            all_phrases = Phrase.objects.exclude(status="auto-validated")
```